### PR TITLE
feat(education)

### PR DESCRIPTION
**Pull request by juliuszfedyk**
education, some refactors, and a component for all small input lists
frontend

### CODE REVIEW

File: src/styles/_global-variables.scss [hunk 1]
```diff
@@ -1,2 +1,3 @@
 $bigfont: 'PT Sans', sans-serif;
 $smallfont: 'Didact Gothic', sans-serif;
+$primary: #ff1493;
```
Comment (drinkbird) [inline src/styles/_global-variables.scss]: `primaryColor` would be more descriptive

File: src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
+@import '../../../styles/global-variables';
+
+$image-button-color: #f0f8ff;
```
Comment (drinkbird) [inline src/app/components/home/home.component.scss]: for consistency we could either use dashes or camelCase for naming sass vars

Comment (juliuszfedyk) [reply src/app/components/home/home.component.scss]: I would say hyphens as scss-lint did suggest it :/

File: src/styles/_global-variables.scss (hunk 1 above)
Comment (juliuszfedyk) [reply src/styles/_global-variables.scss]: you're right

**Pull request merged**